packages/assets-controllers/src/index.ts

```diff
@@ -66,11 +66,17 @@ export type {
 export { TokenListController } from './TokenListController';
 export type {
   Token,
-  TokenRatesConfig,
   ContractExchangeRates,
-  TokenRatesState,
+  TokenRatesControllerState,
+  TokenRatesController,
+  ContractMarketData,
+  TokenRatesControllerGetStateAction,
+  TokenRatesControllerActions,
+  TokenRatesControllerMessenger,
+  TokenRatesControllerStateChangeEvent,
+  TokenRatesControllerEvents,
+  getDefaultTokenRatesControllerState,
 } from './TokenRatesController';
```

Comment:
- Fixes non-types being exported as types.
- nit: Also sorts exports in alphabetical order.

```suggestion
export type {
  ContractExchangeRates,
  ContractMarketData,
  Token,
  TokenRatesControllerActions,
  TokenRatesControllerEvents,
  TokenRatesControllerGetStateAction,
  TokenRatesControllerMessenger,
  TokenRatesControllerState,
  TokenRatesControllerStateChangeEvent,
} from './TokenRatesController';
export {
  getDefaultTokenRatesControllerState, 
  TokenRatesController
} from './TokenRatesController';
```